README.md

```diff
@@ -40,3 +40,7 @@ The source code available in this repository is licensed under the [MIT license]
 5. Open `http://localhost/niceplaces-website/` in your browser.
 
 *TODO: add the instructions to configure the local MySQL database and a Google Maps API key*
+
+Do you want to contribute to Niceplaces?
```

Comment:
Better to use the name of the repository: Nice Places Website

Reply:
Use the name of the repository : Nice Places Website